api/files/index.ts

```diff
@@ -41,13 +41,17 @@ export class SmartlingFilesApi extends SmartlingBaseApi {
     }
 
     async downloadFile(
-        projectId: string, fileUri: string, locale: string, params: DownloadFileParameters
+        projectId: string,
+        fileUri: string,
+        locale: string,
+        responseType: ResponseBodyType | boolean,
```

Comment:
It looks weird. Why not add the 3rd enum value: `PARSE`, `TEXT`, `ARRAY_BUFFER`.
Well, no need to fix here but looks strange.

Reply:
[It's already there](https://github.com/Smartling/api-sdk-nodejs/blob/master/api/base/enum/response-body-type.ts#L2). I think @ChernovDmytro was confused by [this line](https://github.com/Smartling/api-sdk-nodejs/blob/master/api/base/index.ts#L93) in the API SDK base class, but this was only for "backward compatibility" in all the children classes (to not to update them all). [In reality](https://github.com/Smartling/api-sdk-nodejs/blob/master/api/base/index.ts#L138) `ResponseBodyType.TEXT` is the same as using `true`, please remove `| boolean` typing from signature.

Reply:
Agree. It comes from https://github.com/Smartling/api-sdk-nodejs/blob/master/api/base/index.ts#L93 - I only done that to not to make changes in children classes.